src/Init/Data/Order/Lemmas.lean

```diff
@@ -110,8 +151,9 @@ public instance {α : Type u} [LT α] [LE α] [LawfulOrderLT α] :
     intro h h'
     exact h.2.elim h'.1
 
-public instance {α : Type u} [LT α] [LE α] [IsPreorder α] [LawfulOrderLT α] :
-    Std.Irrefl (α := α) (· < ·) := inferInstance
+@[deprecated Asymm.total_not (since := "2025-10-24")]
+public def instIrreflLtOfIsPreorderOfLawfulOrderLT {α : Type u} [LT α] [LE α] [IsPreorder α]
+    [LawfulOrderLT α] : Std.Irrefl (α := α) (· < ·) := inferInstance
```

Comment:
`Asymm.total_not` seems not the right replacement. Did you mean `instIrreflOfAsymm`?